src/sa_web/templates/base.html

```diff
@@ -37,24 +37,26 @@
 
   <!-- Google WebFonts -->
   <link href='//fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-  <link href='https://fonts.googleapis.com/css?family=Alegreya+Sans' rel='stylesheet' type='text/css'>
+  <link href='//fonts.googleapis.com/css?family=Alegreya+Sans' rel='stylesheet' type='text/css'>
 
-  {% if settings.DEBUG %}
+
+  {% compress css %}
   <link rel="stylesheet" href="{{ STATIC_URL }}css/normalize.css">
   <link rel="stylesheet" href="{{ STATIC_URL }}css/leaflet-sidebar.css">
   <link rel="stylesheet" href="{{ STATIC_URL }}css/default.css?deployed_at={{ settings.LAST_DEPLOY_DATE|urlencode:'' }}">
-
   <!-- jQuery date-time picker plugin -->
   <link rel="stylesheet" href="{{ STATIC_URL }}css/jquery.datetimepicker.css">
-  {% else %}
-  <link rel="stylesheet" href="{{ STATIC_URL }}dist/bundle.css"></link>
-  {% endif %}
-  {% compress css %}
   <link rel="stylesheet" type="text/x-scss" href="{{ STATIC_URL }}scss/default.scss" />
   {% endcompress %}
 
   <!-- Flavor's Custom Stylesheet -->
-  <link rel="stylesheet" href="{{ config.static_url }}css/custom.css?deployed_at={{ settings.LAST_DEPLOY_DATE|urlencode:'' }}">
+  <!--  -->
+  <!-- NOTE: -->
+  <!-- The following CSS file is unable to be processed by django compressor
+  because config is accessed through views.py, which requires an HTTP request.
+  However, a request is simply not generated when using manage.py -->
+  <!--  -->
+  <link rel="stylesheet" href="{{ config.static_url }}css/custom.css">
```

Comment:
It looks like we can load the config using Django's template tags, described here: http://stackoverflow.com/a/28741758/1884158

I'm happy to take a look at this solution, hopefully soon...